### DISCUSSION TITLE
Download videos

### DISCUSSION THREAD

**Discussion by emiliekmartinez** [vimeo/player.js#975] [General]
Hey, 

I'm using the javascript SDK to show videos on my website. I'm curious if there's a prop or something I can pass to enable people to download it? I have my video settings set to such but I can't seem to do it from the player.

**Comment by bdougherty** [comment]
Hi, @emiliekmartinez. There is no way to do this currently. The only option would be to direct them to the video page to download from there.

**Reply by emiliekmartinez** [reply]
Is this possible using an "app" inside of my Vimeo account?

**Reply by bdougherty** [reply]
Actually, yes that should be possible. Although I am not 100% certain which account levels have access (that should be in the docs somewhere). Of course that would be outside of the player.
